Code/Tools/TestImpactFramework/Runtime/Common/Code/Source/TestImpactRepoPath.cpp

```diff
@@ -10,6 +10,49 @@
 
 namespace TestImpact
 {
+        RepoPath::RepoPath(const string_type& path) noexcept
```

Comment:
uber nit: Looks like the indentation is slightly off here

Reply:
Will fix